requirements.txt

```diff
@@ -51,3 +51,4 @@ git+git://github.com/HumanCompatibleAI/stable-baselines3@ad902dd5f9d4afeef347897
 magical-il~=0.0.1a4
 dm_control~=0.0.319497192
 git+git://github.com/denisyarats/dmc2gym@6e34d8acf18e92f0ea0a38ecee9564bdf2549076#egg=dmc2gym
+git+https://github.com/rail-berkeley/d4rl@master#egg=d4rl
```

Comment:
I think this also needs `pygame`, `networkx` and `dotmap` (for CARLA specifically).

Reply:
Added!